NuRadioReco/examples/RNO_data/read_data_example/read_rnog.py

```diff
@@ -58,7 +58,10 @@ class EventInfo:
     # Only use runs of a certain run type
     run_types=["physics"],
     # Only use runs with a maximum trigger rate of 1 Hz
-    max_trigger_rate=1 * units.Hz)
+    max_trigger_rate=1 * units.Hz,
+    # This might be necessary to read older files which have an invalid value stored as sampling rate
+    overwrite_sampling_rate=3.2
```

Comment:
@sjoerd-bouma what do you say to this line? I actually do not like it at all. The overwrite is necessary for the test file. I can think of two solutions: Rewrite the interface of the example script which allows to provide a overwrite_sampling_rate via the command line (to be specified in the CI script). Or that we change the reader class and only allow the overwrite to be happening when the sampling rate from the data is actually invalid (or both). What do you think?

Reply:
I had optimistically assumed that was already what the reader did (i.e. overwrite/set the sampling rate only if it's not defined) - if it isn't, I agree that is probably what it should do.

Reply:
Well, I am wondering if there is a reason where we really want to overwrite the sampling rate but I dont know

Reply:
see #641